### PR TITLE
End point for Liveblog datamodel object for dotcom-rendering

### PR DESCRIPTION
**Pull request by shtukas**
## What does this change?

Return an empty dotcom-rendering data model object for live blogs. 

*URL*

```
http://localhost:9000/world/live/2019/apr/23/sri-lanka-bombings-attacks-live-news.json?guui
```

*Before*

Screenshot 2019-05-13 at 13 54 14

*After*

Screenshot 2019-05-13 at 13 54 37

**UPDATE: I've now populated the model as with regular article and also refactored the liveblog controller to hopefully be clearer, so this is okay to review now.**

### CODE REVIEW

File: article/app/controllers/LiveBlogController.scala [hunk 1]
```diff
@@ -14,75 +14,127 @@ import play.api.mvc._
...
+  // Helper methods
+
+  private[this] def render(
+    path: String,
+    page: PageWithStoryPackage,
+    blocks: Blocks
+  )(implicit request: RequestHeader): Future[Result] = {
+    val isMinute = page.isInstanceOf[MinutePage]
 
+    Future.successful {
```
Comment (nicl) [inline article/app/controllers/LiveBlogController.scala]: I wanted to bring all of the routing together for clarity here.

Comment (nicl) [inline article/app/controllers/LiveBlogController.scala]: I couldn't see why this needed to be `Future` before (as that pushes stuff onto a separate EC) but @MatthewJWalls do you know if there was a reason for this?

Comment (MatthewJWalls) [reply article/app/controllers/LiveBlogController.scala]: I think it's fine to be Future. successful. I seem to remember having some trouble getting scala to be happy with the types when we originally split up the article controller and liveblog controller, but there wasn't any concurrency/performance/etc reasons for using a straight Future I think.

**Comment by shtukas** [comment]
Checked locally. Perfect. Thanks @nicl

**Review by MatthewJWalls — APPROVED** [review]
Make sure you test all endpoints locally at least! Code looks good.

**Pull request merged**